src/AddressRegistry/StreetName/Events/AddressPositionWasChanged.cs

```diff
@@ -0,0 +1,81 @@
+namespace AddressRegistry.StreetName.Events
+{
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Newtonsoft.Json;
+
+    [EventTags(EventTag.For.Edit, EventTag.For.Sync)]
+    [EventName(EventName)]
+    [EventDescription("De adrespositie werd gewijzigd.")]
+    public class AddressPositionWasChanged : IStreetNameEvent, IHasAddressPersistentLocalId
+    {
+        public const string EventName = "AddressPositionWasChanged"; // BE CAREFUL CHANGING THIS!!
+
+        [EventPropertyDescription("Objectidentificator van de straatnaam aan dewelke het adres is toegewezen.")]
+        public int StreetNamePersistentLocalId { get; }
+
+        [EventPropertyDescription("Objectidentificator van het adres.")]
+        public int AddressPersistentLocalId { get; }
+
+        [EventPropertyDescription("Geometriemethode van de adrespositie. Mogelijkheden: DerivedFromObject of AppointedByAdministrator.")]
+        public GeometryMethod GeometryMethod { get; }
+
+        [EventPropertyDescription("Specificatie van het object dat voorgesteld wordt door de adrespositie. Mogelijkheden: Municipality, Street, Parcel, Lot, Stand, Berth, Building, BuildingUnit, Entry of RoadSegment.")]
+        public GeometrySpecification GeometrySpecification { get; }
```

Comment:
Verwijderen we roadsegment niet?

Reply:
@ArneD , mag dit weg of dient dit te blijven bestaan omwille van data uit Crab?

Reply:
I would remove it because once we're here => new business logic => no roadsegment available (for the moment).

Reply:
@ArneD roadSegment lijkt nog een gebruikte waarde zijn in staging:

```
SELECT top(100) Id, JsonData
  FROM [address-registry-events].[AddressRegistry].[Messages]
  where 
  Position >= 122254637
  and [Type] = 'AddressWasMigratedToStreetName'
  and JSON_VALUE(JsonData, '$.geometrySpecification') = 'roadSegment'
```